st2api/tests/unit/controllers/v1/test_alias_execution.py

```diff
@@ -322,15 +322,8 @@ def test_match_and_execute_list_action_param_str_cast_to_list(self):
         self.assertEqual(resp.status_int, 201)
 
         result = resp.json["results"][0]
-        live_action = result["execution"]["liveaction"]
         action_alias = result["actionalias"]
-
         self.assertEqual(resp.status_int, 201)
-        self.assertTrue(isinstance(live_action["parameters"]["array_param"], list))
```

Comment:
LiveAction is no longer an embedded document in the actionexecution table.   If users are using that output then it will not longer exist.  Do we need to make a new controller for liveaction so it can be retrieved?